ui/spec_helper.rb

```diff
@@ -16,6 +16,9 @@
 
 # Site Prism URL defaults to SSL in production
 ENV['BASE_URL'] ||= 'http://localhost:3000'
+ENV['PRIVACY_URL']||= 'http://localhost:3000/static/privacy_policy'
```

Comment:
What is the benefit to having an env per path?

Reply:
Currently, it's a placeholder - as there is no obvious way to navigate to the /static/ pages.